NesCollectorWebUI/NesCollectorWebUI/Startup.cs

```diff
@@ -33,11 +37,22 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-            //bad way of adding connection string
-            //TODO: fix this later somehow!
-            var connectionString = "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=nescollector;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
+            //repository layer  injection
+            //services.AddScoped<IGameRepository, MockGameRepository>();
+            //services.AddScoped<IUserGameRepository, MockUserGameRepository>();
+            //services.AddScoped<IUserRepository, MockUserRepository>();
+            //services.AddScoped<IWishlistRepository, MockWishlistRepository>();
+            services.AddScoped<IGameRepository, EFCoreGameRepository>();
```

Comment:
Make sure you move all this `Scoped` items to private methods to avoid listing to many thing in the `ConfigureServices` method